packages/node-runtime-worker-thread/webpack.config.js

```diff
@@ -42,7 +42,8 @@ const config = {
     'mongodb-client-encryption': 'commonjs2 mongodb-client-encryption',
     kerberos: 'commonjs2 kerberos',
     snappy: 'commonjs2 snappy',
-    interruptor: 'commonjs2 interruptor'
+    interruptor: 'commonjs2 interruptor',
+    'os-dns-native': 'commonjs2 os-dns-native',
```

Comment:
Yep, looks good! I know that I said that we should externalize it here (and it's totally the correct thing to do) but what I forgot about is due the way we connect this runtime in Compass strictly speaking it's not even required as we connect the app through dataService first (where os-dns-native will already do all the work) and then we pass the driver connection options to this module

Reply:
Right -- but currently, we're still getting a "Missing optional dependency" warning fro mongosh in Compass because of this. I think it might be nice to avoid that :)